jail.lua

```diff
@@ -0,0 +1,9 @@
+beerchat.jail_list = {}
+
+if nil == beerchat.jail_channel_name or "" == beerchat.jail_channel_name then
+	beerchat.jail_channel_name = "wail"
```

Comment:
gut-feeling why I choose wail and not jail, also because when I started I did not call it jailing user but was looking for a short word for 'go time out in corner' -> wail at wall.
I have no problem whatever it is called :)
There is also a reason to call it other than jail in user facing side: less confusion with old jail process.